src/traces/splom/index.js

```diff
@@ -220,30 +220,16 @@ function hoverPoints(pointData, xval, yval, hovermode) {
     var id, ptx, pty, dx, dy, dist, dxy;
     var minDist = maxDistance;
 
-    if(hovermode === 'x') {
```

Comment:
OK sure, that'll make people happier until we do a better job with defaults - just make a comment about why we're omitting `'x'`, so we don't forget about this issue.

Reply:
Removed for now.